src/Unity/Helper/Unity.Data.cs

```diff
@@ -10,23 +13,77 @@ public partial class Unity
     public Module UnityPlayer { get; private set; }
     public SceneManager Scenes { get; private set; }
 
+    private string _dataFolder;
+    public string DataFolder
+    {
+        get
+        {
+            string directory = Path.GetDirectoryName(MainModule.FilePath);
+            string folderName = _dataFolder ?? Path.GetFileName(MainModule.FilePath)[..^4];
+
+            return Path.Combine(directory, folderName);
```

Comment:
please check whether this output is correct on other games. it does not appear to me that this will result in the output you expect.

Reply:
It works for me even if I don't specify it, so you'll have to be more specific with what you mean.

Reply:
I have tried on other games and it all works